.wasser/config.yaml

```diff
@@ -55,10 +55,11 @@ routines:
         sudo systemctl start libvirtd
     - name: run microos image on vagrant
       command: |
-        cd aquarium/tests/vagrant
-        ./setup.sh foo 2>&1
-        cd setups/foo
-        vagrant up --no-destroy-on-error --no-parallel 2>&1
+        cd aquarium
+        source venv/bin/activate
+        ./tools/aqrdev create foo 2>&1
+        ./tools/aqrdev start foo --conservative 2>&1
+        deactivate
```

Comment:
You do not need to deactivate it since, this channel is dropped, every new command needs own env

Reply:
alright, will adjust.

Reply:
yeah... for illustration it looks okay to leave it here to correspond steps in FZTH

Reply:
Does this mean this command does not have a virtualenv without running `./setup-dev.sh` first?